include/amber/amber.h

```diff
@@ -15,12 +15,16 @@
 #ifndef AMBER_AMBER_H_
 #define AMBER_AMBER_H_
 
+#include <map>
 #include <string>
+#include <vector>
 
 #include "amber/result.h"
 
 namespace amber {
 
+using ShaderMap = std::map<std::string, std::vector<uint32_t>>;
```

Comment:
`ShaderMap` is not intuitive to me. Could you please add a comment to explain meaning of its key and value?

Reply:
done

Reply:
nit: represntation --> representation

Reply:
done